test/rocc/README

```diff
@@ -0,0 +1,28 @@
+This tests basic RoCC accelerator functionality using the default dummy_rocc
+accelerator included with spike. You may use this as a template for writing
+bare-metal software for RoCC accelerators.
+
+# Key Files
+* crt.S: Initializes the system and sets up the basic runtime. This eliminates the need for an OS.
+* rocc.h: These are helper macros for constructing RoCC instructions
+* syscalls.c: This implements a subset of the C standard library. We do not
+  include all of C, so you'll need to implement whatever you need that isn't
+  there.
+* link.ld: This instructs the linker to setup the binary as a standalone
+  bootable image on the bare-metal hardware without any OS.
+* rocc-test.c: This is our actual benchmark. For the most part, a new project
+  would only need to modify this file.
+
+# Usage
+Build with:
+    make
+
+Run on Spike with:
+    spike --extension=dummy_rocc rocc-test.riscv
+
+Run in Marshal:
+    ./marshal build test/rocc.json
+    ./marshal launch -s test/rocc.json
+
+or:
```

Comment:
What is the difference between this option and the previous option?

Reply:
I'll clarify in the README. The first one runs the test but doesn't validate the outputs. The 'test' command automatically builds and validates outputs.

Reply:
Validate against what?

Reply:
This readme assumes you've read the documentation of FireMarshal. I don't want to repeat everything here. This is just a minimal description. It should explain how building and running a RoCC workload is different than other workloads. These one-liners are here just so you can copy/paste if you forgot the exact syntax.

Reply:
So it might be better to remove the last line (test) in order to not confuse anyone (if we're expecting people to have read the full documentation before hand, then they will know the `test` feature exist, and they don't need it in the readme).

Reply:
I took another stab at the README. Let me know if it's more clear now.